src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/GoalEntity.kt

```diff
@@ -42,12 +42,14 @@ class GoalEntity(
   val goalOrder: Int,
 
   @Column(name = "plan_uuid")
-  val planUuid: UUID,
+  var planUuid: UUID? = null,
```

Comment:
they can't be saved without one, but I've assumed here that an incoming Goal from the UI won't have one?

Reply:
I missed this, I think it should be 
`val planUuid: UUID.randomUUID()`

Reply:
No, it can't be random - a goal must be associated with a Plan.

In the API I've populated the goal.planUuid field from the API path. Are we saying that the UI essentially passes it in two places for a single call?